update.sh

```diff
@@ -2,4 +2,4 @@
 
 UPDATE=1 nix run nixpkgs#gnumake nixpkgs#curl -c make update --keep-going || true
 
-nix flake update --update-input released-nixpkgs || true
+nix flake update --recreate-lock-file || true
```

Comment:
Why this? Seems better to update the minimum necessary.

Reply:
You are right. I should have just added nix-pills to be also updated.